meteor/server/api/blueprints/context/context.ts

```diff
@@ -545,41 +553,22 @@ export class TimelineEventContext extends RundownContext implements ITimelineEve
 	}
 }
 
-export class AsRunEventContext extends RundownContext implements IAsRunEventContext {
-	public readonly asRunEvent: Readonly<IBlueprintAsRunLogEvent>
-
+export class AsRunRundownEventContext extends RundownContext implements IAsRunRundownEventContext {
```

Comment:
Semantically, is this still _AsRun..._ - I'm fine with this conceptually - this is the as run timing information for this rundown or part. Its just with the collection going away ...

Reply:
Ive renamed these to be a bit clearer now